contribs/gmf/src/controllers/abstract.js

```diff
@@ -93,41 +82,52 @@ gmf.AbstractController = function(
   /**
    * @type {ngeo.GetBrowserLanguage}
    */
-  this.getBrowserLanguage = ngeoGetBrowserLanguage;
+  this.getBrowserLanguage = $injector.get('ngeoGetBrowserLanguage');
 
   /**
    * @type {ngeo.StateManager}
    */
-  this.stateManager = ngeoStateManager;
+  this.stateManager = $injector.get('ngeoStateManager');
 
   /**
    * @type {angular.Scope}
    */
-  this.scope = $scope;
+  this.$scope = $scope;
```

Comment:
I would say yes, i've reviewed other work in that direction. It's better to keep knowing what comes from Angular or not, with the `$`, what are others opinion ?